tests/builder/help.rs

```diff
@@ -964,6 +999,71 @@ OPTIONS:
     ));
 }
 
+#[test]
+#[cfg(all(feature = "wrap_help"))]
+fn possible_value_wrapped_help() {
+    #[cfg(feature = "unstable-v4")]
+    static WRAPPED_HELP: &str = "test 
+
+USAGE:
+    test [OPTIONS]
+
+OPTIONS:
+    -h, --help
+            Print help information
+
+        --possible-values <possible_values>
+            Possible values:
+                - name: Long enough help message to clearly
+                          warrant wrapping
+                - second
+
+        --possible-values-with-new-line <possible_values_with_new_line>
+            Possible values:
+                - long enough name to trigger new line:
+                    Long enough help message to clearly warrant
+                    wrapping
+                - second
+";
+    #[cfg(not(feature = "unstable-v4"))]
+    static WRAPPED_HELP: &str = r#"test 
+
+USAGE:
+    test [OPTIONS]
+
+OPTIONS:
+    -h, --help                                                             Print help information
+        --possible-values <possible_values>                                [possible values: name, second]
+        --possible-values-with-new-line <possible_values_with_new_line>    [possible values: "long enough name to trigger new line", second]
```

Comment:
Should this maybe also switch to next-line-help? Not really related to this PR, but seams to produce unexpected results.

Reply:
Unsure; would need more digging into to decide what the behavior should be

Reply:
The reason this does not wrap is, that `--possible-values-with-new-line <possible_values_with_new_line>` is already wider than the terminal.

This is therefor correct behavior following the current implementation here:
https://github.com/clap-rs/clap/blob/cad5430922813ba6086d66ac90e2ab295b360c73/src/output/help.rs#L562

Reply:
More so I think we can defer this out of this PR and focus the conversation in a separate issue

Reply:
Still technically wrong thinking about it, because it actually would have fit, if this would not consider the space behind as well. 

IMO it should be `self.term_w >= longest + 8` instead of 12.